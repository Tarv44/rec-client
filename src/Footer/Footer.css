footer {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    width: 100%;
    background-color: #5C18F0;
    color: white;
}