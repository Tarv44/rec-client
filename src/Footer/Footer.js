import React from 'react';
import './Footer.css'

export default function Footer() {
    return (
        <footer>
            <p>&copy; 2021 Jonah Tarver</p>
        </footer>
    )
}