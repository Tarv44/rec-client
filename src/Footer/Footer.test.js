import React from 'react'
import ReactDom from 'react-dom'
import renderer from 'react-test-renderer'
import Footer from './Footer'


describe('----- Footer -----', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDom.render(<Footer />, div)
        ReactDom.unmountComponentAtNode(div);
    })
    it('renders the UI as expected', () => {
        const comp = renderer
            .create(<Footer />)
            .toJSON()
        expect(comp).toMatchSnapshot()
    })
})