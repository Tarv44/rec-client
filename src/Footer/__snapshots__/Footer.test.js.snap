// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`----- Footer ----- renders the UI as expected 1`] = `
<footer>
  <p>
    Â© 2021 Jonah Tarver
  </p>
</footer>
`;
